docs/cncf_conformance.md

```diff
@@ -0,0 +1,167 @@
+gcp_compute_subnetwork_private = "10.64.4.0/22"
+# Inbound Master Access
+admin_cidr = "0.0.0.0/0"
+
+# Uncomment the line below if you want short living cheap cluster for testing
+#gcp_scheduling_preemptible = "true"
+```
+
+Now, launch the DC/OS cluster by running
+
+```
+$ make get-cli launch-dcos setup-cli
+```
+
+This command will:
+
+1. Download the `dcos` CLI and `kubectl` to your machine.
+1. Provision the necessary infrastructure in GCP and install DC/OS.
+1. Setup the `dcos` CLI to access the newly created DC/OS cluster.
+
+As part of the last step, your browser will open and ask you to login with
+a Google, GitHub or Microsoft account. Choose an option and copy the resulting
+OpenID token to the shell where you ran the abovementioned command.
+
+## Installing `dcos-kubernetes`
+
+To install `dcos-kubernetes` in the newly created DC/OS cluster run
+
+```
+$ dcos package install --yes --options=./resources/options-ha.json kubernetes
+```
+
+Wait until all tasks are running before proceeding. You can track installation
+progress using
+
+```
+$ watch dcos kubernetes plan show deploy
+```
+
+When installation is successful you will see the following output:
+
+```
+deploy (serial strategy) (COMPLETE)
+├─ etcd (serial strategy) (COMPLETE)
+│  ├─ etcd-0:[peer] (COMPLETE)
+│  ├─ etcd-1:[peer] (COMPLETE)
+│  └─ etcd-2:[peer] (COMPLETE)
+├─ apiserver (parallel strategy) (COMPLETE)
+│  ├─ kube-apiserver-0:[instance] (COMPLETE)
+│  ├─ kube-apiserver-1:[instance] (COMPLETE)
+│  └─ kube-apiserver-2:[instance] (COMPLETE)
+├─ kubernetes-api-proxy (parallel strategy) (COMPLETE)
+│  └─ kubernetes-api-proxy-0:[install] (COMPLETE)
+├─ controller-manager (parallel strategy) (COMPLETE)
+│  ├─ kube-controller-manager-0:[instance] (COMPLETE)
+│  ├─ kube-controller-manager-1:[instance] (COMPLETE)
+│  └─ kube-controller-manager-2:[instance] (COMPLETE)
+├─ scheduler (parallel strategy) (COMPLETE)
+│  ├─ kube-scheduler-0:[instance] (COMPLETE)
+│  ├─ kube-scheduler-1:[instance] (COMPLETE)
+│  └─ kube-scheduler-2:[instance] (COMPLETE)
+├─ node (parallel strategy) (COMPLETE)
+│  ├─ kube-node-0:[kube-proxy] (COMPLETE)
+│  ├─ kube-node-0:[coredns] (COMPLETE)
+│  ├─ kube-node-0:[kubelet] (COMPLETE)
+│  ├─ kube-node-1:[kube-proxy] (COMPLETE)
+│  ├─ kube-node-1:[coredns] (COMPLETE)
+│  ├─ kube-node-1:[kubelet] (COMPLETE)
+│  ├─ kube-node-2:[kube-proxy] (COMPLETE)
+│  ├─ kube-node-2:[coredns] (COMPLETE)
+│  └─ kube-node-2:[kubelet] (COMPLETE)
+├─ public-node (parallel strategy) (COMPLETE)
+└─ mandatory-addons (serial strategy) (COMPLETE)
+   ├─ mandatory-addons-0:[kube-dns] (COMPLETE)
+   ├─ mandatory-addons-0:[metrics-server] (COMPLETE)
+   ├─ mandatory-addons-0:[dashboard] (COMPLETE)
+   └─ mandatory-addons-0:[ark] (COMPLETE)
+```
+
+## Accessing the Kubernetes API
+
+In order to access the Kubernetes API from outside the DC/OS cluster, run
+
+```
+$ make kubectl-tunnel
+```
+
+Now, in a different shell, run `kubectl get nodes`. The output should be similar
```

Comment:
In a different shell, check that `kubectl` is properly configured to communicate with your new cluster by running `kubectl get nodes`. The output should look something like: